probe-rs/src/debug/variable.rs

```diff
@@ -361,6 +422,8 @@ pub struct Variable {
     pub byte_size: Option<u64>,
     /// If this is a subrange (array, vector, etc.), is the ordinal position of this variable in that range
     pub member_index: Option<i64>,
+    /// If this is a bit field type, we need to store the offset and bit length.
+    pub bitfield: Option<BitField>,
```

Comment:
Does this need a new field in this struct? I would prefer another solution, there are too many fields which are only used in some cases already. Maybe a variant of the type?

Reply:
The comment already says this is a bitfield type, so it seems natural to me that this would just be a different type.
For example, what would it mean if a `VariableType::Struct` has this set?